src/mode/remapper.ts

```diff
@@ -120,8 +125,17 @@ class Remapper {
       }
 
       vimState.isCurrentlyPerformingRemapping = false;
-
       return true;
+    } else {
+      // Check to see if a remapping could potentially be applied when more keys are received
+      for (let remap of this._remappings) {
```

Comment:
or do you mean break off the function into a isRemappingPotential() function that takes the current keystrokes as an input?